Index: com/rapplogic/xbee/api/PacketParser.java
===================================================================
--- com/rapplogic/xbee/api/PacketParser.java	(revision 322)
+++ com/rapplogic/xbee/api/PacketParser.java	(working copy)
@@ -140,7 +140,7 @@
 			// length of api structure, starting here (not including start byte or length bytes, or checksum)
 			this.length = new XBeePacketLength(msbLength, lsbLength);
 
-			log.debug("packet length is " + ByteUtils.formatByte(length.getLength()));
+			log.debug(java.lang.String.format("packet length is %s [0x%04X]", length.getLength(), length.getLength()));
 			
 			// total packet length = stated length + 1 start byte + 1 checksum byte + 2 length bytes
 			
